be/CMakeLists.txt

```diff
@@ -489,6 +489,58 @@ if ("${USE_STAROS}" STREQUAL "ON")
        )
 endif()
 
+# LLVM
+# TODO(yueyang): Adjust libraries during the implementation of JIT.
+set (LLVM_LIBRARIES
+    LLVMRuntimeDyld
+    LLVMBitstreamReader
+    LLVMOption
+    LLVMAsmPrinter
+    LLVMProfileData
+    LLVMAsmParser
+    LLVMOrcTargetProcess
+    LLVMExecutionEngine
+    LLVMBinaryFormat
+    LLVMDebugInfoDWARF
+    LLVMObjCARCOpts
+    LLVMCodeGen
+    LLVMMCDisassembler
+    LLVMSupport
+    LLVMJITLink
+    LLVMCFGuard
+    LLVMInstrumentation
+    LLVMIRReader
+    LLVMCore
+    LLVMTarget
+    LLVMMC
+    LLVMAnalysis
+    LLVMGlobalISel
+    LLVMScalarOpts
+    LLVMTargetParser
+    LLVMDemangle
+    LLVMRemarks
+    LLVMDebugInfoCodeView
+    LLVMOrcShared
+    LLVMOrcJIT
+    LLVMTextAPI
+    LLVMBitWriter
+    LLVMBitReader
+    LLVMObject
+    LLVMTransformUtils
+    LLVMSelectionDAG
+    LLVMMCParser
+)
+if ("${CMAKE_BUILD_TARGET_ARCH}" STREQUAL "x86" OR "${CMAKE_BUILD_TARGET_ARCH}" STREQUAL "x86_64")
+    set(${LLVM_LIBRARIES} ${LLVM_LIBRARIES} LLVMX86CodeGen LLVMX86Desc LLVMX86Info)
+elseif ("${CMAKE_BUILD_TARGET_ARCH}" STREQUAL "aarch64")
+    set(${LLVM_LIBRARIES} ${LLVM_LIBRARIES} LLVMAArch64CodeGen LLVMAArch64Desc LLVMAArch64Info)
+endif()
+
+foreach(lib IN ITEMS ${LLVM_LIBRARIES})
+    add_library(${lib} STATIC IMPORTED GLOBAL)
+    set_target_properties(${lib} PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/llvm/lib/lib${lib}.a)
```

Comment:
need double check if this is always `llvm/lib` or possibly `llvm/lib64` which is platform dependent.

Reply:
According to the installation documentation of LLVM, the default installation location for LLVM libraries is `llvm/lib`. To install them in `llvm/lib64`, specifying the "64" suffix is necessary.Additionally, I tested it on an M1 Pro Mac, and the installation location was indeed `llvm/lib`.

![image](https://github.com/StarRocks/starrocks/assets/40347749/233ed256-2327-40a1-a25e-a91d9f6cc3dd)

https://llvm.org/docs/CMake.html#:~:text=Extra%20suffix%20to%20append%20to%20the%20directory%20where%20libraries%20are%20to%20be%20installed.%20On%20a%2064%2Dbit%20architecture%2C%20one%20could%20use%20%2DDLLVM_LIBDIR_SUFFIX%3D64%20to%20install%20libraries%20to%20/usr/lib64.